### DISCUSSION TITLE
miniPaint as Gimp?

### DISCUSSION THREAD

**Discussion by ghost** [viliusle/miniPaint#356] [Q&A]
Hi.

I was thinking of making a version of miniPaint as Gimp online.

#### ui/ux
![image](https://user-images.githubusercontent.com/125943841/221423525-37d67847-7a36-4a04-ae6f-09fe6c0c25e6.png)

What do you think about the idea of ​​miniPaint looking like GIMP?

**Comment by viliusle** [comment]
You can create new theme, but this design would require much more features, including not only CSS, but also JS.
I am not sure about it, but I do not think that GIMP has best design. And priority for this is not high, because current design is not that bad.

But if you think that you want to do this, you can try, but letting you know that there will be lots of small features that you will need to implement with JS like new icons, new buttons and so on.
